impl/src/bin/cargo-raze.rs

```diff
@@ -126,7 +146,10 @@ fn real_main(options: &Options, cargo_config: &mut Config) -> CliResult {
     GenMode::Remote => {
       // Create "remote/" if it doesn't exist
       if fs::metadata("remote/").is_err() {
-        fs::create_dir("remote/").map_err(failure::Error::from)?;
+        match fs::create_dir("remote/") {
```

Comment:
Any reason to prefer the match over `map_err(|e| Err(anyhow::format_err!(...)))`?